util/src/main/java/io/kubernetes/client/util/ModelMapper.java

```diff
@@ -242,9 +252,21 @@ public GroupVersionKind(String group, String version, String kind) {
       this.kind = kind;
     }
 
-    private String group;
-    private String version;
-    private String kind;
+    private final String group;
```

Comment:
We should really use `GroupVersionKind` objects for storing these values.

Reply:
rebased and fixed